lifelines_transform/main.py

```diff
@@ -2,29 +2,37 @@
 import logging
 import json
 import os
-import sys
 import time
 
 import download
 from lifelines_transform import __version__
 from transform import Transform
 from upload import Upload
 
-startTime = time.time()
 
+execution_start_time = time.time()
 project_dir = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../'))
 
+# Use UTC time for all logging.
+logging.Formatter.converter = time.gmtime
+
 log = logging.getLogger('transform')
-FORMAT = '[%(levelname)s] %(asctime)-15s %(message)s'
-formatter = logging.Formatter(FORMAT)
+log.setLevel(logging.DEBUG)
```

Comment:
do we want to have the level set to DEBUG in production ?

Reply:
Niet echt; ik zal kijken of ik het configurabel kan maken